bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/AccordionImpl.java

```diff
@@ -40,26 +48,60 @@
     public final static String RESOURCE_TYPE = "core/wcm/components/accordion/v1/accordion";
 
     @ValueMapValue(optional = true)
-    private String expandedItem;
+    private boolean singleExpansion;
 
     @ValueMapValue(optional = true)
-    private String headingType;
+    private String[] expandedItems;
 
-    private String expandedItemName;
+    @ValueMapValue(optional = true)
+    private String headingElement;
+
+    @ScriptVariable
+    private Style currentStyle;
+
+    private String[] expandedItemNames;
+
+    /**
+     * The {@link com.adobe.cq.wcm.core.components.internal.Utils.Heading} object for the HTML element
+     * to use for accordion headers.
+     */
+    private Utils.Heading heading;
```

Comment:
Shouldn't we have a similar approach in the Tabs component then?

Reply:
Tabs is a slightly different concept, it isn't recommended to use heading level elements there:
https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html

The accordion design pattern does recommend them: https://www.w3.org/TR/wai-aria-practices/#accordion